epigraphhub/analysis/epistats.py

```diff
@@ -34,7 +34,13 @@ def prevalence(pop_size: int, positives: int, a: float = 1, b: float = 1) -> obj
     return st.beta(pa, pb)
 
 
-def inf_pos_prob_cases_hosp(df: pd.DataFrame) -> az.data.inference_data.InferenceData:
+def inf_pos_prob_cases_hosp(
+    df: pd.DataFrame,
+    alpha: float = 0.5,
+    beta: float = 0.5,
+    draws: int = 2000,
+    tune: int = 500,
+) -> az.data.inference_data.InferenceData:
     """
     This function compute the posterior probability distribution for the prevalence of cases and the probability of hospitalization over time.
```

Comment:
After the refactoring is done, I  will start a specific issue  (or issues) to generalized the functions in the epistats module.